coder_sniffer/DrupalPractice/Test/ProjectDetection/ProjectUnitTest.php

```diff
@@ -1,79 +0,0 @@
-<?php
-
-/**
- * Tests that project and version detection works.
- */
-class ProjectUnitTest extends PHPUnit_Framework_TestCase
```

Comment:
This test seems to be useful, any reason why this is being deleted?

Reply:
As mentioned in the previous PR the ProjectUnitTest used functionality in phpcs that no longer exists - I haven't looked into how to re-instate it at this point - how about if we created an issue to re-instate it and do that in there since this PR is about upgrading and, well, this PR is a monster as it is.

Reply:
Thoughts?

Reply:
OK, that makes sense. We can figure out how to deal with this in a followup.

Instead of removing the test, let's keep it in for now and mark the test as skipped. I would also feel more comfortable if the followup is already created and mentioned in a comment. This will help to make sure it will not be forgotten in the future.

Example:
```
// @see https://drupal.org/node/[#followup issue]
$this->markTestSkipped('This test relies on code that has been removed in PHP_CodeSniffer 3.x.');
```

Reply:
Yep, sounds good. Will get back to this in the next couple days.